docs/how-to-setup-mobile-app-locally.md

```diff
@@ -0,0 +1,140 @@
+# Setup freeCodeCamp's mobile app locally
+
+## System requirements
+
+These are the different requirements needed to contribute to the mobile application for the different operating systems. These specifications are based of running a Flutter project and two installed emulators.
+
+#### Windows
+
+- Microsoft Windows 7/8/10 (64-bit)
```

Comment:
Seeing as Microsoft no longer supports `7`, I would assume this should be the case:

```suggestion
- Microsoft Windows 8/10 (64-bit)
```